help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,159 @@
+
+Once the destination has been created, you will be shown the audience activation flow. The following walks through how to activate audiences using the Real-Time destination.
+
+### Map attributes and identities {#map}
+
+The next step is mapping source identifiers to the Magnite device_id identifier.
+
+- You can add as many mappings as you need by selecting **[!UICONTROL Add new mapping]**.
+
+In this example using the Real-Time destination, we're mapping any rows that contain a generic deviceId source identifier to the Magnite device_id target field. When you\'re done, click Next.
+
+![map desired data fields to the device_id field](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-field-mapping.png)
+
+![Be sure to set mapping ids to all activated audiences, or set NONE if no mapping id is present](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-mappingid.png)
+
+You must now configure a Start date (mandatory), an End date (optional), and a Mapping ID for each audience.
+
+**Mapping ID**
+
+- Use the **[!UICONTROL Mapping ID]** field when an audience has a pre-existing Segment ID previously known to Magnite.
+
+- To add a **[!UICONTROL Mapping ID]** to an audience, click each audience row individually, and enter data in the right-hand column (see image above). If you do not want to add a Mapping ID, please enter NONE into the Mapping ID field. 
+
+Select **[!UICONTROL Next]** and finalize the activation flow.
+
+
+`![](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-review.png)
+
+## Exported data / Validate data export {#exported-data}
+
+Once your audiences have been uploaded, you may validate your audiences have been created and uploaded correctly using the following steps:
+
+- In 95% of cases, audiences will be delivered to Magnite Streaming in under 10 minutes. The actual receipt and processing of the events within Magnite Streaming depends on the shared data volume.
+
+- Post-ingest, segments are expected to appear in Magnite Streaming within a few minutes and can be applied to a deal. You can confirm this by looking up the segment ID that was shared during the activation steps in the Adobe Experience Platform.
+
+## Activate the same audiences through the Magnite Streaming: Batch destination
+Audiences shared with Magnite Streaming using the  Real-Time destination will also need to be shared using the [Magnite Streaming: Batch](magnite-batch.md) destination. When configured correctly, segment names in the Magnite Streaming UI are updated to reflect those used in the Adobe Experience Platform post-daily update.
+
+Finally, if a Batch destination has not been configured for your integration, set it up now via [this document](magnite-batch.md)
```

Comment:
is there a rule as to which of the two destinations (batch & real-time) should be set up first? and I imagine the same audiences and same mappings should be selected for both destinations. is that correct?

Reply:
This is correct. Both destinations need to be used together. This was addressed with Vlad's comments.